regparser/grammar/unified.py

```diff
@@ -72,6 +74,22 @@ def appendix_section(match):
                     | atomic.upper_p)
        ).setParseAction(appendix_section).setResultsName("appendix_section"))
 
+# "the" appendix implies there's only one, so it better be appendix A
+section_of_appendix_to_this_part = (
+    atomic.section_marker
+    + atomic.upper_roman_a.setResultsName("appendix_section")
+    + Literal("of the appendix to this part").setResultsName("appendix").setParseAction(lambda: 'A')
```

Comment:
Yeah, there's a comment right above it actually. It's because it's "the appendix" so there's only one. Sadly, I discovered during my parsing of X that an appendix can be named any damn thing that the writer wants, so perhaps this isn't the safest assumption. I've yet to see it violated though.